vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/Spreadsheet.java

```diff
@@ -3985,15 +3990,7 @@ protected SpreadsheetClientRpc getRpcProxy() {
         return clientRpc;
     }
 
-    /*
-     * (non-Javadoc)
-     *
-     * @see com.vaadin.ui.AbstractComponent#beforeClientResponse(boolean)
-     */
-    // @Override
-    public void beforeClientResponse(boolean initial) {
```

Comment:
The root problem here was that this method never got called. In V8, it was automatically called because the method was overridden from AbstractComponent. For Flow Spreadsheet, I needed to add an explicit invocation.